app/assets/stylesheets/base/_fonts.scss

```diff
@@ -1,5 +1,5 @@
 // This file contains the base fonts for the application.
 
-@import url(https://fonts.googleapis.com/css?family=Lato:100,400,300);
+@import url(https://fonts.googleapis.com/css?family=Roboto:100,400,300);
 
-$font-family: 'Lato';
+$font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', sans-serif;
```

Comment:
I saw a footnote on google font lato that a previous version has the extended Latin set, which includes Vietnamese. We can keep Lato I think, if only to support Vietnamese. But in the long run we'll need to support language-specific base fonts because you're not going to be able to have one font cover everything. We can do a short term fix but we'll have to architect once there's a need for Japanese, Chinese, Thai or some language that Lato doesn't cover.